app/views/layouts/application.html.haml

```diff
@@ -7,8 +7,11 @@
     = csrf_meta_tags
 
   %body
+    %h1.title CSE Force request System 
     = render 'layouts/header'
-    %h1.title CSE Force request System
+    - unless session[:uin].nil?
+      = render 'layouts/login'
```

Comment:
I can't see any login layout file. I think you forgot to commit it by mistake. Please check